configs/manager/halfstack.toml

```diff
@@ -22,7 +22,6 @@ ssl-enabled = false
 #ssl-cert = "/etc/backend.ai/ssl/apiserver-fullchain.pem"   # env: BACKNED_SSL_CERT
 #ssl-privkey = "/etc/backend.ai/ssl/apiserver-privkey.pem"  # env: BACKNED_SSL_KEY
 # ipc-base-path = "/tmp/backend.ai/ipc"
-heartbeat-timeout = 10.0
```

Comment:
To let developers know the existence of the parameters, let's keep the parameter with 40s. default for now. :D

Reply:
Done.